src/apig_wsgi.py

```diff
@@ -48,6 +49,36 @@ def handler(event, context):
     return handler
 
 
+def v2_to_multivalue(obj):
+    multi_value = {}
+    for key, value in obj.items():
+        multi_value[key] = value.split(",")
+    return multi_value
+
+
+def get_event_from_version(event):
+    if event.get("version") == "1.0":
+        return event
+
+    headers = event.get("headers", {})
+    multi_value_headers = {}
+    for header, value in headers.items():
+        multi_value_headers[header] = value.split(",")
```

Comment:
this is unused

Reply:
Good catch! I missed this. I'll tear it out.